docs/rules/symbol-description.md

```diff
@@ -1,17 +1,29 @@
 # Require/Disallow Symbol Description (symbol-description)
 
-The `Symbol` constructor may have optional description:
+The `Symbol` function may have optional description:
 
 ```js
 var foo = Symbol("foo");
 
 var someString = 'bar';
 var bar = Symbol(someString);
+```
+
+
+Using `description` promotes easier debugging since when symbol is casted to string the description is used:
```

Comment:
You can cast symbol to a string, just not implicitly via concatenation. The browser is just doing this casting:

```js
const s = Symbol('description');
String(s); // "Symbol('description')"
s + ''; // TypeError
```

So the description was correct, just the example wasn't.